test/unit/controllers/adhoc_payment/submit_amount_controller_test.js

```diff
@@ -75,9 +77,7 @@ describe('adhoc payment submit-amount controller', function () {
     })
 
     it('should add a relevant error message to the session \'flash\'', () => {
-      expect(session.flash).to.have.property('genericError')
-      expect(session.flash.genericError.length).to.equal(1)
-      expect(session.flash.genericError[0]).to.equal(`<h2>Use valid characters only</h2> Choose an amount in pounds and pence using digits and a decimal point. For example “10.50”`)
+      expect($('.generic-error').text()).to.include(`Choose an amount in pounds and pence using digits and a decimal point. For example “10.50”`)
```

Comment:
We should add classes to the html tags rather than asserting on text (i18n) but I think is like this everywhere, I think is fine but just to something keep in mind and maybe ask for next PRs